### ISSUE TITLE
Build failed due to use of deprecated Android v1 embedding.

### ISSUE THREAD

**Issue by tenSunFree** [mkobuolys/flutter-design-patterns#41]
Currently using flutter_windows_2.10.1-stable.

Build failed due to use of deprecated Android v1 embedding.
<img width="482" alt="001" src="https://user-images.githubusercontent.com/23090085/153705357-6520db16-a59b-4027-bab3-6aa78e8fabc2.png">

You can try to delete android:name="io.flutter.app.FlutterApplication".
<img width="484" alt="002" src="https://user-images.githubusercontent.com/23090085/153705397-0d09bb2b-334f-4b94-85d9-7980d37c558d.png">

**Comment by martin-braun** [comment]
Nah, don't delete that, instead put the applicationName variable into it like so: [Baseflow/flutter-permission-handler@`e9664eb` (#791)](https://github.com/Baseflow/flutter-permission-handler/pull/791/commits/e9664eb179971e09be7e5717ebbc7520a68cffdf)
